Src/LinearSolvers/MLMG/AMReX_MLNodeLaplacian.cpp

```diff
@@ -1140,6 +1161,18 @@ MLNodeLaplacian::interpolation (int amrlev, int fmglev, MultiFab& fine, const Mu
 
     const iMultiFab& dmsk = *m_dirichlet_mask[amrlev][fmglev];
 
+    bool regular_coarsening = true; int idir = 2;
+    if (amrlev == 0 and fmglev > 0) {
+        regular_coarsening = mg_coarsen_ratio_vec[fmglev] == mg_coarsen_ratio;
+	IntVect ratio = (amrlev > 0) ? IntVect(2) : mg_coarsen_ratio_vec[fmglev];
```

Comment:
The test on amrlev is unnecessary.  Also the indentation seems to be off.  If this is because of tabs, please use four white spaces instead of tab.